apps/system/style/notifications/notifications.css

```diff
@@ -146,15 +147,20 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
   border: 0;
-  background: none;
+  border-radius: 0;
   color: #008eab;
+  background-color: none;
```

Comment:
@etiennesegonzac okk really sorry :) I saw a Stack Overflow answer which said there is no practical difference between using `background-color` and `background`. 
Extremely sorry